packages/edit-site/src/index.js

```diff
@@ -89,16 +86,6 @@ export function reinitializeEditor( target, settings ) {
 			showListViewByDefault: false,
 		} );
 
-		// Check if the block list view should be open by default.
-		if (
-			select( preferencesStore ).get(
-				'core/edit-site',
-				'showListViewByDefault'
-			)
-		) {
-			dispatch( editSiteStore ).setIsListViewOpened( true );
-		}
```

Comment:
This was moved to the action that toggles the "edit" canvas mode that way, we open the list view by default when the "edit mode is triggered"